main.cpp

```diff
@@ -14,16 +16,27 @@ int main(int argc, char *argv[])
             push(el);
         }
     }
+    /*---------------------------------------------*/
+    Logger log("log_example.txt"); // класс Logger
+    /*---------------------------------------------*/
     //queue.begin = new List;//List(0);
     int el;
     int change_list = 1; //выбор списка
     int change = 1; //выбор действий
+    int change_class = 1; //выбор класса
+    int change_queue =1; //выбор действя для queue (class)
+    myQueue* myQu = new myQueue;
+    myDequeue* myDequ = new myDequeue;
     unsigned int s_time;
-    while((change_list>=1) && (change_list<=2)) {
+    while((change_list>=1) && (change_list<=3)) {
+        /*---------------------------------------------*/
+        log.print("It's work!"); // класс Logger
+        /*---------------------------------------------*/
         //выбор списка
         cout << "Change:" << endl;
         cout << "1 - Queue" << endl;
-        cout << "2 - My list" << endl;
```

Comment:
Queue & deque should be used only from Class implementation. Previous code isn't required anymore.